blueprints/app/files/tests/test-helper.js

```diff
@@ -1,8 +1,7 @@
-import resolver from './helpers/resolver';
-import {
-  setResolver
-} from 'ember-qunit';
-import { start } from 'ember-cli-qunit';
+import { start } from 'ember-qunit';
```

Comment:
Can we put this at the bottom of the imports (order of use). Just a nit-pick.

Reply:
Sure

Reply:
Done 🕺